modules/react/collection/lib/ListBox.tsx

```diff
@@ -53,6 +53,7 @@ export const ListBox = createContainer('ul')({
   },
 })<ListBoxProps>(({height, maxHeight, ...elemProps}, Element, model) => {
   // TODO figure out what style props should go to which `Box`
+  const renderedItems = useListRenderItems(model, elemProps.children) as React.ReactNode;
```

Comment:
i wonder if we need to change the return type of `useListRenderItems` so that we don't have to do this every time we use this hook